docs/mods.json

```diff
@@ -619,6 +619,11 @@
     "name": "Sends Stats to DBL",
     "section": "Other Stuff"
   },
+  "send_stats_to_botlist.me_MOD.js": {
```

Comment:
Lets keep this alphabetically sorted

Reply:
So send_stats_to_botlist.me should be above send_stats_to_dbl